gnd/src/main/res/layout/record_list_item.xml

```diff
@@ -14,94 +14,106 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
+<layout xmlns:tools="http://schemas.android.com/tools"
+  xmlns:android="http://schemas.android.com/apk/res/android">
 
-<FrameLayout
-  xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:background="?android:attr/selectableItemBackground"
-  android:clickable="true">
-  <RelativeLayout
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_marginHorizontal="@dimen/record_list_item_horizontal_margin"
-    android:layout_marginVertical="@dimen/record_list_item_horizontal_vertical">
-    <TextView
-      android:id="@+id/user_name"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:textAppearance="@style/RecordListText.CreatorName"
-      tools:text="Ground User"/>
-    <TextView
-      android:id="@+id/separator_dot1"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_toRightOf="@id/user_name"
-      android:text=" · "
-      />
-    <TextView
-      android:id="@+id/last_modified_date"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_toRightOf="@id/separator_dot1"
-      android:textAppearance="@style/RecordListText.Timestamp"
-      tools:text="Sept 25, 2018"/>
-    <TextView
-      android:id="@+id/separator_dot2"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_toRightOf="@id/last_modified_date"
-      android:text=" · "
-      android:textAppearance="@style/RecordListText.Separator"/>
-    <TextView
-      android:id="@+id/last_modified_time"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_toRightOf="@id/separator_dot2"
-      android:textAppearance="@style/RecordListText.Timestamp"
-      tools:text="4:25 PM"/>
+  <data>
+
+    <variable
```

Comment:
These two variables look a lot like a ViewModel. Should we move them into a new class, `RecordListItemViewModel`?